CIP-0120/README.md

```diff
@@ -0,0 +1,487 @@
+
+This proposal aims to describe a standardized technical form for the Cardano
+Constitution to enhance the accessibility and safety of the document.
+
+> **Note:** This proposal only covers the technical form of the constitution,
+> this standard is agnostic to the content of the constitution.
+
+## Motivation: why is this CIP necessary?
+
+CIP-1694 defines the on-chain anchor mechanism used to link the off-chain
+Constitution document to on-chain actions.
+This mechanism was chosen due to its simplicity and cost effectiveness,
+moving the potentially large Cardano constitution off-chain,
+leaving only a hash digest and URI on-chain.
+This is the extent to which CIP-1694 outlines the Cardano Constitution:
+CIP-1694 does not provide suggestions around hashing algorithm,
+off-chain storage best practices or use of rich text styling.
+
+By formalizing the form of the constitution and its iterations,
+we aim to promote its longevity and accessibility.
+This is essential to ensure the effectiveness of the CIP-1694 governance model.
+
+This standard will impact how Ada holders read the Constitution but
+the main stakeholders for this are the tool makers who wish to read,
+render and write a constitution.
+
+### Safety
+
+Without describing best practices for the form and handling of the constitution,
+ we risk the constitution document being stored in an insecure manner.
+By storing the constitution on a decentralized platform,
+we can ensure its immutability and permissionless access.
+This is a step to improve the longevity and accessibility of each constitution
+iteration.
+
+### Interoperability
+
+By defining a file extension and formatting rules for the constitution we
+ensure that tooling reading and writing the constitution will be interoperable.
+Furthermore we aim to make the role of constitution iteration comparison tools
+easier, by minimizing formatting and style changes between iterations.
+This will reduce compatibility issues between tools,
+promoting the accessibility of the constitution.
+
+### Usability
+
+Rich text formatting greatly enhances the readability of text,
+especially in large complex documents.
+Without the ability to format text, it could easily become cumbersome to read,
+negatively effecting the accessability of the Cardano constitution.
+
+## Specification
+
+The following specification SHOULD be applied to all constitution iterations.
+This standard could be augmented in the future via a separate CIP which aims to
+replace this one.
+
+### Terminology
+
+#### Capitalisation of key terms
+
+To avoid unnecessary edits, and therefore checksum changes,
+constitution authors MUST follow the following standard English capitalisation
+rules unless a translation language indicates otherwise:
+
+`Constitution` vs. `constitution`
+
+- The Constitution in effect — either the "initial" one or any updated version — is unique and therefore capitalised ("Constitution") as a proper noun.
```

Comment:
```suggestion
- The Constitution in effect — either the "initial" one or any new constitution — is unique and therefore capitalised ("Constitution") as a proper noun.
```
The changes @Hornan7 proposed today in https://github.com/cardano-foundation/CIPs/pull/847 would make sense to update here as well: since, especially if any constitution updates are hashed completely (rather than storing or hashing their difference) that every "updated" constitution is in fact a _new_ constitution.

Reply:
sure, makes sense!